app/views/components/show.html.erb

```diff
@@ -6,11 +6,11 @@
 
 <h3 class="govuk-heading-m">Signing Certificates (Enabled)</h3>
 <% @certificates =  @component.enabled_signing_certificates %>
-<%= render "certificates/certificates_table" %>
+<%= render "certificates/certificates_table" , status: 'disable' %>
 
 <h3 class="govuk-heading-m">Signing Certificates (Disabled)</h3>
 <% @certificates =  @component.disabled_signing_certificates %>
```

Comment:
reassigning a ivar inside a view is bit smelly. Could we pass this as a parameter to the partial like `status`?
```
 <%= render "certificates/certificates_table" , 
     status: 'enable', 
     certificates: @component.enabled_signing_certificates
 %>
```